Add explicit pin directions to examples

brentru opened the pull request:
While digitalio defaults to input, it's very important for the `IRQ` and `CS` pins to be set as inputs, IRQ is expected to be pulled up at the end of the `send_packet()` method. 

Patch: 
* Adding explicit pin directions to examples in case the default changes and for clarity.

jerryneedell commented:
CS is an output 
IRQ is an input

jerryneedell commented:
I'd rather see IRQ set to INPUT in adafruit_tinylora.py. The user just provides the pins

brentru commented:
I've set the irq as input and cs as output from the driver instead of the example, added `self._cs` as well, passed into the bus device, removed directions in examples/
